### PR TITLE
[BOD-270] Fix optional ids and textarea export in the design system

### PR DESCRIPTION
**Pull request by saruba**
BOD-270

### CODE REVIEW

File: .storybook/main.ts [hunk 1]
```diff
@@ -52,13 +52,6 @@ const config: StorybookConfig = {
       imageRule['exclude'] = /\.svg$/i
     }
 
-    if (config.resolve) {
-      config.resolve.alias = {
-        ...config.resolve.alias,
-        '@': resolve(__dirname, '../src'),
-      }
-    }
-
```
Comment (verochan) [inline .storybook/main.ts:61]: sad

**Review by verochan — COMMENTED** [review]
@saruba I've been reviewing again these components, and I wonder if their `name` property shouldn't be too `name={identifier}` instead of `name={name}. What do you think?

I've added a commit where I fixed a typo on the Changelog and I've taken advantage of it to clarify better the changes, as we were repeating the params were optional on the two latest versions.

**Comment by saruba** [comment]
> @saruba I've been reviewing again these components, and I wonder if their `name` property shouldn't be too `name={identifier}` instead of `name={name}`. What do you think?
> 
> I've added a commit where I fixed a typo on the Changelog and I've taken advantage of it to clarify better the changes, as we were repeating the params were optional on the two latest versions.

We can't use the `identifier` for `name` because we could never put a `name` different from the `id`, the `id` always replaces the name. We can create an `identifier` prop name or use the `id` in `id` and `name` but we have to remove the `name` from props.

**Comment by verochan** [comment]
> > @saruba I've been reviewing again these components, and I wonder if their `name` property shouldn't be too `name={identifier}` instead of `name={name}`. What do you think?
> > I've added a commit where I fixed a typo on the Changelog and I've taken advantage of it to clarify better the changes, as we were repeating the params were optional on the two latest versions.
> 
> We can't use the `identifier` for `name` because we could never put a `name` different from the `id`, the `id` always replaces the name. We can create an `identifier` prop name or use the `id` in `id` and `name` but we have to remove the `name` from props.

I think this will complicate the logic behind identifier / id / name. Let's stick with the current changes and if in the future we detect some pain, we'll think about it.

**Pull request merged**